source/get-started/bootable-usb.rst

```diff
@@ -33,184 +22,61 @@ Burn the |CL| image onto a USB drive
-      /dev/sdb      7.5G General  UDisk                    usb    disk                              
-      └─/dev/sdb1   7.5G                                          part Microsoft Basic Data         /run/media/clear/CENA_X64FRE
-      /dev/sdc       15G          Patriot_Memory           usb    disk                              
-      └─/dev/sdc1    15G                                          part                              /run/media/clear/U
-
-   .. note::
-
-      Some Linux distros may automatically mount a USB drive when it is plugged in.
-
-#. If the USB drive you want to use is mounted, it must be umounted before
-   burning an image onto it.  Use the :command:`umount` command followed by
-   the device identifier/partition. For example, to unmount all of the 
-   ``/dev/sdc`` partitions:
+#. Launch balenaEtcher.
 
-   .. code-block:: bash
+.. figure:: /_figures/bootable-usb/balenaEtcher_Start.PNG
 
-      sudo umount /dev/sdc*
-
-#. Burn the image onto the USB drive. This example burns an image onto ``/dev/sdc``.  
-   The device name of the USB may vary.
-
-   .. code-block:: bash
-
-      sudo dd if=./clear-[version number]-live-[desktop | server].iso of=/dev/sdc oflag=sync bs=4M status=progress
-
-.. caution::
-
-   Not fully unmounting the USB drive before burning an image could cause
-   file system checksum errors in it. If this happens, burn the image again,
-   ensuring all the USB drive partitions are unmounted first.
-
-.. _bootable-usb-mac:
-
-Create a bootable USB drive on macOS\*
-**************************************
-
-* Make sure you completed all `Prerequisites`_.
-
-* :ref:`verify-mac` on macOS.
-
-Burn the |CL| image onto a USB drive
-====================================
-
-.. caution::
-
-   Burning an image formats the USB drive and destroys all pre-existing
-   content.  Back up your data before proceeding.
-
-#. Open a Terminal window.
+#. Press :guilabel:`Select Image`.
 
 #. Change directory to where the image resides.
 
-#. Plug in a USB drive and get its identifier:
-
-   .. code-block:: bash
-
-      diskutil list
-
-   This lists available disks and their partitions, as shown in Figure 1.
-
-   .. figure:: /_figures/bootable-usb/bootable-usb-mac-01.png
-      :scale: 100 %
-      :alt: Get USB drive identifier
-
-      Figure 1: macOS - Get USB drive identifier
-
-#. Unmount the USB drive identified in the previous step. For example, to unmount /dev/disk2. The device name of the USB may vary.
+#. Open the Image.
 
-   .. code-block:: bash
+.. figure:: /_figures/bootable-usb/balenaEtcher_ImgaeSlect.PNG
 
-      diskutil umountDisk /dev/disk2
+#. Plug in the USB drive.
 
-#. Burn the image onto the drive using the :command:`dd` command.
-   This example uses `./`, your current directory, and it shows how to burn
-   an image onto `/dev/disk2`:
+#. Identify the USB drive using the :guilabel:`Change` under the middel icon.
+   .. note::
```

Comment:
I would upgrade this to a warning and reiterate they should be careful to select the right USB drive.